barretenberg/cpp/src/barretenberg/vm2/common/tagged_value.cpp

```diff
@@ -128,7 +128,7 @@ uint8_t get_tag_bytes(ValueTag tag)
     case ValueTag::U128:
         return get_tag_bits(tag) / 8;
     case ValueTag::FF:
-        return 32;
+        return 0;
```

Comment:
These values are more potentially more useful in future constraints if they are set o 0